services/viva-ms/helpers/vivaAdapterRequestClient.js

```diff
@@ -25,7 +25,11 @@ async function sendVivaAdapterRequest({ endpoint, method, body = undefined }) {
     if (requestError.response) {
       // The request was made and the server responded with a
       // status code that falls out of the range of 2xx
-      throw requestError.response;
+      throw {
+        vadaResponse: requestError.response.data,
+        status: requestError.response.status,
+        headers: requestError.response.headers,
+      };
```

Comment:
This is because the requestError object is somewhat over verbose and the relevant error info lives in the specified attributes. Found it more easy like this to track when error occure.

Reply:
Point noted. Will fix this in later task.